src/plugins/intel_cpu/src/nodes/random_uniform.cpp

```diff
@@ -240,19 +428,34 @@ inline void runPhilox(uint64_t key, uint64_t counter, uint64_t n, uint32_t* res)
     uint32_t* counter_32 = reinterpret_cast<uint32_t*>(&counter);
     uint32_t* n_32 = reinterpret_cast<uint32_t*>(&n);
 
-    for (size_t i = 0lu; i < ROUNDS_NUMBER; i++) {
-        calculateRound(key_32, counter_32, n_32);
-        if (i < ROUNDS_NUMBER - 1)
-            raiseKey(key_32);
-    }
+    // Loop unwarping for better performance
```

Comment:
Do you have performance numbers for this?

Reply:
No, the idea is to remove the JMP instruction caused by the loop, assuming the optimization settings when building the OV bin file are insufficient. The performance gain is most likely marginal

Reply:
Please measure it first. It is usually cheaper to jump than loading a long stack. You may run out of cache.